arch/arm64/src/imx9/Kconfig

```diff
@@ -82,8 +82,11 @@ menu "USDHC"
 
 config IMX9_USDHC
 	bool
-	default n
 	select ARCH_HAVE_SDIO_PREFLIGHT
+	select FAT_DMAMEMORY
+	select FAT_FORCE_INDIRECT
```

Comment:
I'm afraid he is not the right place to enable it. Because users could use USDHC with other FS for instance.
@xiaoxiang781216 @pkarashchenko any suggestions to fix it?

Reply:
check FATFS before select FAT_DMAMEMORY/FAT_FORCE_INDIRECT?

Reply:
Thanks for the comments!

Now I see that this PR doesn't really make sense. Those flags are only needed if FAT is used on eMMC/SD cards. I don't think that there is a nice automatic way to do that. What Xiao suggested is closest, but not quite accurate either (there could be FAT on some other device, in which case these flags wouldn't be needed).

I will just add the configs into my own boards defconfigs, and cancel this PR.

Reply:
@jlaitine something that you can do to help users is adding a failsafe lock in the board file related to USDHC initialization using #ifdefs : if IMX9_USDHC and VFAT is enabled but FAT_DMAMEMORY/FAT_FORCE_INDIRECT is not enabled, force an #error with a informative message to users enable it.